bin/download_minister.py

```diff
@@ -0,0 +1,165 @@
+            if row not in existing_list:
+                # case: new minister in choices, add new minister to existing list
+                existing_list[row] = choices[row]
+
+        return set_status(existing_list)
+    # both lists do not exist
+    else:
+        return False
+
+
+def download_xml_from_source():
+    """
+    Read XML from external source to generate JSON choices for ministers
+    """
+    choices = {}
+    xml_tree = {}
+
+    # collect responses for both languages
+    for langCode in lang_codes:
+        # url = 'https://www.ourcommons.ca/Parliamentarians/' + langCode + '/ministries/Export?output=XML'
+        url = 'http://localhost/export-' + langCode + '.xml'
+        response = requests.get(url, stream=True)
+
+        # if valid response then save as xml tree
+        if response.status_code == 200:
+            xml_tree[langCode] = etree.fromstring(response.content)
+
+            for row in xml_tree[langCode].xpath('Minister'):
+                # gather name and position
+                name = row.xpath('PersonOfficialFirstName/text()')[0] + ' ' + \
+                       row.xpath('PersonOfficialLastName/text()')[0]
+                position = {
+                    'title_' + langCode: row.xpath('Title/text()')[0],
+                    'start_date': row.xpath('FromDateTime/text()')[0],
+                    'end_date': ''.join(row.xpath('ToDateTime/text()'))
+                }
+                # if name already exist in json
+                if name in choices:
+                    # add translation
+                    if langCode not in choices[name]:
+                        choices[name].update([
+                            (langCode, row.xpath('PersonOfficialLastName/text()')[0] + ', '
+                             + row.xpath('PersonOfficialFirstName/text()')[0]
+                             + ' (' + row.xpath('PersonShortHonorific/text()')[0] + ')'),
+                            ('honorific_title_' + langCode, row.xpath('PersonShortHonorific/text()')[0]),
+                            ('positions_' + langCode, [position]),
+                            ])
+                    else:
+                        # append position
+                        choices[name]['positions_' + langCode].append(position)
+
+                # new name in json
+                else:
+                    choices[name] = {
+                        langCode: row.xpath('PersonOfficialLastName/text()')[0] + ', '
+                        + row.xpath('PersonOfficialFirstName/text()')[0]
+                        + ' (' + row.xpath('PersonShortHonorific/text()')[0] + ')',
+                        'honorific_title_' + langCode: row.xpath('PersonShortHonorific/text()')[0],
+                        'precedence': row.xpath('OrderOfPrecedence/text()')[0],
+                        'positions_' + langCode: [position],
+                    }
+        response.close()
+    return choices
+
+
+def set_status(list):
```

Comment:
please don't shadow builtin names like "list" with variable names (It's also confusing to name a dict "list")